build/ci/evergreen.yml

```diff
@@ -282,6 +284,20 @@ functions:
           export GOROOT="${go_root}"
           export PATH="./bin:$GOROOT/bin:$PATH"
           ./bin/snyk test --org=cloud
+  "install pykmip":
+    - command: subprocess.exec
+      type: setup
+      params:
+        add_to_path:
+          - ${go_bin}
+          - ../../bin
+        working_dir: /src/github.com/mongodb/mongocli/build/ci/kmip
+        binary: ./install.sh
+        args: ["-h", "hosts.json"]
+        env:
+          XDG_CONFIG_HOME: ${workdir}
+          KMIP_CA: ${logs_decrypt_kmip_ca}
+          KMIP_CERT: ${logs_decrypt_kmip_cert}
```

Comment:
the kmip server (same as the cli) needs the same CA, cert and db (where the key encryption keys are stored) to be able to decrypt existing logs so generating new certs won't work.